pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -699,6 +722,36 @@ func TestEventListenerValidate_error(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "missing label and namespace selector",
+		el: bldr.EventListener("name", "namespace",
+			bldr.EventListenerSpec(
+				bldr.EventListenerTriggerGroup(v1alpha1.EventListenerTriggerGroup{
+					Name: "my-group",
+					Interceptors: []*v1alpha1.TriggerInterceptor{{
+						Ref: v1alpha1.InterceptorRef{
+							Name: "cel",
+						},
+						Params: []v1alpha1.InterceptorParams{{
+							Name:  "filter",
+							Value: test.ToV1JSON(t, "has(body.repository)"),
+						}},
+					}},
+				}))),
+	}, {
+		name: "triggerGroup requires interceptor",
+		el: bldr.EventListener("name", "namespace",
+			bldr.EventListenerSpec(
```

Comment:
One request: Could we use regular structs instead of the builders? We have been trying to get reduce/remove test builder usage for a while -- See https://github.com/tektoncd/pipeline/issues/3178 for some discussion

Reply:
Aha, I wasn't aware that this was the goal. I will refactor

Reply:
Aha, I wasn't aware that this was the goal. I will refactor